docs/tutorials/clustered-deployment-ssl.rst

```diff
@@ -298,12 +298,12 @@ Now that we have all of the Docker dependencies installed, we can create a Docke
 
 .. _clustered_quickstart_compose_ssl:
 
-Docker Compose: Setting Up a Three Node CP Cluster with SSL
+Docker Compose: Setting Up a Three Node Confluent Platform Cluster with SSL
```

Comment:
not a huge deal, but you might want to extend the underline here. sphinx annoyingly complains with warnings if the underline is shorter than the title above it.